external-docs/docs/guides/configuration/qube-config.md

```diff
@@ -121,56 +125,60 @@ There are several configuration options available across column types except obs
 | `from_existing`  | The uri of the resource for reuse/extension (Optional)                                                                    | *none*                                                                           |
 | `definition_uri` | A uri of a resource to show how the column is created/managed (i.e. a uri of a PDF explaining a list of units) (Optional) | *none*                                                                           |
 
-The `from_existing` value when set provides the basis of linked data; it allows csvcubed to generate additional RDF-hints to allow users to discover how the `tidy_data.csv` links to other data semanticly.
+The `from_existing` value when set provides the basis of linked data; it allows csvcubed to generate additional RDF-hints to allow users to discover how the `tidy_data.csv` links to other data semantically.
 
-## Dimension configuration
+## Dimension Configuration
 
 | **field name**      | **description**                                                                                                                                                                                                      | **default value**                                                                |
 |---------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------|
 | `type`              | The type of the column (Required)                                                                                                                                                                                    | *dimension*                                                                      |
+| `from_template`     | Use a [column template](templates.md)                                                                                                                                                                                | *none*                                                                           |
 | `label`             | The title of the column (Optional)                                                                                                                                                                                   | The capital case of the header in the csv file with spaces replacing underscores |
```

Comment:
I agree with what you mentioned in your notes about making the casing consistent for acronyms like CSV or URI. This is a place where an inconsistency like that exists, I found that it is written as "csv" within table text, but outside of the tables it is always written as "CSV". A very small (maybe meaningless) inconsistency but maybe still worth looking at.

Reply:
Added to new ticket (#706)